webservices/common/models/itemized.py

```diff
@@ -109,20 +114,32 @@ class ScheduleB(BaseItemized):
     recipient_zip = db.Column(db.String)
     disbursement_type = db.Column('disb_tp', db.String)
     disbursement_description = db.Column('disb_desc', db.String)
-    disbursement_date = db.Column('disb_dt', db.Date)
+    disbursement_date = db.Column('disb_dt', db.TIMESTAMP(timezone=False))
     disbursement_amount = db.Column('disb_amt', db.Numeric(30, 2))
+    election_type = db.Column('election_tp', db.String)
+    election_type_full = db.Column('election_tp_desc', db.String)
+    entity_type = db.Column('entity_tp', db.String)
+    entity_type_desc = db.Column('entity_tp_desc', db.String)
     back_reference_transaction_id = db.Column('back_ref_tran_id', db.String)
     back_reference_schedule_id = db.Column('back_ref_sched_id', db.String)
+    memo_code = db.Column('memo_cd', db.String)
+    memo_code_full = db.Column('memo_cd_desc', db.String)
+    memo_text = db.Column(db.String)
     national_committee_nonfederal_account = db.Column('national_cmte_nonfed_acct', db.String)
-    election_type = db.Column('election_tp', db.String)
-    election_type_full = db.Column('election_tp_desc', db.String)
-    record_number = db.Column('record_num', db.Integer)
-    report_primary_general = db.Column('rpt_pgi', db.String)
-    receipt_date = db.Column('receipt_dt', db.Date)
+    original_sub_id = db.Column('orig_sub_id', db.Integer)
+    fec_election_type_desc = db.Column('fec_election_tp_desc', db.String)
+    fec_election_year = db.Column('fec_election_tp_year', db.String)
+    #These are deprecated
+    #record_number = db.Column('record_num', db.Integer)
+    #report_primary_general = db.Column('rpt_pgi', db.String)
+    #receipt_date = db.Column('receipt_dt', db.Date)
```

Comment:
I definitely think they should be removed, I left them more as a reminder to @LindsayYoung as two what will be deprecated, in correspondence with the api release documentation.  Which could also be done with a diff on the table schemas.